doc/findinfiles.rst

```diff
@@ -1,27 +1,58 @@
 :orphan:
 
-#############
-Find in Files
-#############
+####
+Find
+####
 
-The **Find in Files** pane allows to you perform full-text, recursive content search in a user-defined scope, with advanced features to filter your results.
+The **Find** pane allows you to search the content of files in a user-defined location, with advanced features to filter your results.
 
 .. image:: images/find_in_files/find_in_files_inprogress.png
    :alt: Spyder Find in Files panel, with search results shown per-file
 
 |
 
 
+
+===================
+Using the Find pane
+===================
+
+To search for text in the Find pane, write it in the top bar of the pane and press the search button.
+This will allow you to view and navigate through all the occurrences of your search text in your working directory.
+You can expand or contract the search results to view the results on each file.
+Clicking on any of these matches, will automatically open the file showing where the text was found.
+
+[GIF showing search and cliking file]
+
+If you want to change the scope of your search, select another directory, project or file in the :guilabel: 'Search in' menu.
+The locations that you select for your search will be stored in the list so you can access them easily.
+To erase all of these locations, select the "clear the list" option from the dropdown menu in the "Search in" field.
```

Comment:
```suggestion
To erase all of these saved directories, select the :guilabel:`Clear the list" option from the dropdown menu in the :guilabel:`Search in` field.
```

Avoided repetition and made it a little more precise; used reST directives where appropriate, used accurate capitalization as shown in the UI